brigade/templates/events.html

```diff
@@ -68,7 +68,7 @@ <h3>
   </div>
 </section>
 
-<section id="workshops-and-meetings" class="slab slab-gray">
+<section id="workshops-and-meetings" class="slab gray-bg-slab">
```

Comment:
`gray-bg-slab` 🤢 i'm excited to get rid of these bad class names, but good to be consistent for now :+1:

Reply:
Yeah, can't wait to bid these classes good riddance.